### DISCUSSION TITLE
Can’t view most repos

### DISCUSSION THREAD

**Discussion by mobilemind** [community/community#3911] [Mobile] [answered]
Unable to open or view 14 out of my 27 repos for the last 12 hours using iOS app. 

Error is: “Something happened. Try again.”

Same repos view fine in Safari and clone with no errors on command line. About 6 of them were changed an hour before the issue (rename default branch to main). The other half haven’t been changed in over 9 months. 

Sign-out of app. Uninstall app. Revoke app credentials on web. Reinstall app. Reauthorize app. Still the same issue. 

Issue replicates on iPhone XS, iOS 14.5.1 and on iPad Pro, iPadOS 14.5.1.

**Comment by mobilemind** [comment]
Issue still happens with latest release of GitHub Mobile 1.14.0 (35987939).

Able to replicate the issue 100% of the time with the same repos.

Zero work-around available on Apple devices.

Same repos fine in desktop clients, GitHub.com, and terminal command line interface.

**Comment by Iron-Ham** [comment]
Hi @mobilemind thank you for bringing this to our attention. Is this occurring on an Enterprise account, or your personal GitHub account? Are these repos part of an organization that may require SAML?

**Comment by mobilemind** [comment]
Happens on a personal GitHub account. No SAML or SSO involved. 

See also Ticket 1140134

https://support.github.com/ticket/1140134

**Reply by mobilemind** [reply]
Resolved as of GitHub iOS 1.15.0

**Comment by mobilemind** [comment, marked as answer]
Resolved as of GitHub for iOS 1.15.0.
